### PR TITLE
Debian implementation

### PR DESCRIPTION
**Pull request by bersace**

### CODE REVIEW

**Comment by bersace** [comment]
@jpic

File: lxchost/init.sls [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+{% set network = salt['pillar.get']('lxchost:network', '10.0.4.0') -%}
+{% set range = salt['pillar.get']('lxchost:range', '10.0.4.1,10.0.4.250') -%}
+
+lxc_pkgs:
+  pkg.installed:
+    - pkgs:
+      - aufs-tools
+      - bridge-utils
+      - dnsmasq
+      - iptables
```
Comment (jpic) [inline lxchost/init.sls:15]: I think we should have a separate iptables formula and depend on it.

File: lxchost/init.sls [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+{% set netmask = salt['pillar.get']('lxchost:netmask', '255.255.255.0') -%}
+{% set network = salt['pillar.get']('lxchost:network', '10.0.4.0') -%}
+{% set range = salt['pillar.get']('lxchost:range', '10.0.4.1,10.0.4.250') -%}
+
+lxc_pkgs:
+  pkg.installed:
+    - pkgs:
+      - aufs-tools
+      - bridge-utils
+      - dnsmasq
```
Comment (jpic) [inline lxchost/init.sls:14]: I think we should have a dnsmasq formula and depend on it.

File: lxchost/init.sls [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      - iptables
+      - lxc
+      - netcat-traditional
+      - sudo
+    - reload_modules: True
+    - reload_modules: True
```
Comment (jpic) [inline lxchost/init.sls]: double

File: lxchost/init.sls [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+lxc-post-up:
+  file.managed:
+    - name: /etc/network/if-up.d/lxchost
```
Comment (jpic) [inline lxchost/init.sls]: I think this should be a pillar variable.

File: lxchost/init.sls [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    - mode: 0750
+    - source: salt://lxchost/post-up.sh
+    - template: jinja
+    - defaults:
+        iface: {{ iface }}
+        network: {{ network }}
+
+lxc-post-down:
+  file.managed:
+    - name: /etc/network/if-post-down.d/lxchost
```
Comment (jpic) [inline lxchost/init.sls]: I think this should be a pillar variable.

File: lxchost/init.sls [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    - source: salt://lxchost/post-down.sh
+    - template: jinja
+    - defaults:
+        iface: {{ iface }}
+        network: {{ network }}
+
+disable_dnsmasq:
+  service.dead:
+    - name: dnsmasq
+    - enable: False
```
Comment (jpic) [inline lxchost/init.sls:23]: I think this belongs to a dnsmasq formula

File: lxchost/init.sls [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+etclxc:
+  file.directory:
+    - name: /etc/lxc
```
Comment (jpic) [inline lxchost/init.sls]: I think this should be a pillar variable.

File: lxchost/init.sls [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+
+conf_lxc_dnsmasq:
+  file.managed:
+    - name: /etc/lxc/dnsmasq.conf
```
Comment (jpic) [inline lxchost/init.sls]: I think this should be a pillar variable.

File: lxchost/init.sls (hunk 4 above)
Comment (bersace) [inline lxchost/init.sls]: Why ?

File: lxchost/init.sls (hunk 2 above)
Comment (bersace) [inline lxchost/init.sls:14]: Why ?

File: lxchost/init.sls (hunk 1 above)
Comment (bersace) [inline lxchost/init.sls:15]: Why ?

File: lxchost/init.sls (hunk 5 above)
Comment (bersace) [inline lxchost/init.sls]: Why ?

File: lxchost/init.sls (hunk 7 above)
Comment (bersace) [inline lxchost/init.sls]: Why ?

File: lxchost/init.sls (hunk 8 above)
Comment (bersace) [inline lxchost/init.sls]: Why ?

**Pull request merged**